packages/react/src/components/Toolbar/ToolbarItem.tsx

```diff
@@ -194,8 +210,8 @@ class ToolbarItem extends UIComponent<WithAsProp<ToolbarItemProps>, ToolbarItemS
     const renderedItem = (
       <ElementType
         {...accessibility.attributes.root}
-        {...applyAccessibilityKeyHandlers(accessibility.keyHandlers.root, unhandledProps)}
         {...unhandledProps}
+        {...applyAccessibilityKeyHandlers(accessibility.keyHandlers.root, unhandledProps)}
```

Comment:
should we document that `applyAccessibilityKeyHandlers` needs to be spread after `unhandledProps` on a component?

Reply:
@layershifter have you documented it when added this? frankly, I don't remember why 😄

Reply:
It's done for correctly overriding `onKeyDown` on slots. It should be applied after because `unhandledProps` can contain `onKeyDown` from user and is handled by UTs in `isConformant()`

Reply:
added as a comment